packages/wdio-browserstack-service/src/launcher.ts

```diff
@@ -271,38 +271,8 @@ export default class BrowserstackLauncherService implements Services.ServiceInst
         // remove accessibilityOptions from the capabilities if present
         this._updateObjectTypeCaps(capabilities, 'accessibilityOptions')
 
-        if (this._accessibilityAutomation) {
-            const scannerVersion = await createAccessibilityTestRun(this._options, this._config, {
-                projectName: this._projectName,
-                buildName: this._buildName,
-                buildTag: this._buildTag,
-                bstackServiceVersion: BSTACK_SERVICE_VERSION,
-                buildIdentifier: this._buildIdentifier,
-                accessibilityOptions: this._options.accessibilityOptions
-            })
-
-            if (scannerVersion) {
-                process.env.BSTACK_A11Y_SCANNER_VERSION = scannerVersion
-            }
-            BStackLogger.debug(`Accessibility scannerVersion ${scannerVersion}`)
-        }
-
-        if (this._options.accessibilityOptions) {
-            const filteredOpts = Object.keys(this._options.accessibilityOptions)
-                .filter(key => !NOT_ALLOWED_KEYS_IN_CAPS.includes(key))
-                .reduce((opts, key) => {
-                    return {
-                        ...opts,
-                        [key]: this._options.accessibilityOptions?.[key]
-                    }
-                }, {})
-
-            this._updateObjectTypeCaps(capabilities, 'accessibilityOptions', filteredOpts)
-        } else if (isAccessibilityAutomationSession(this._accessibilityAutomation)) {
-            this._updateObjectTypeCaps(capabilities, 'accessibilityOptions', {})
-        }
-
-        if (this._options.testObservability) {
+        const shouldSetupPercy = this._options.percy || (isUndefined(this._options.percy) && this._options.app)
```

Comment:
percy auto-enabled needs to be handled?

Reply:
The condition after || is for auto enabling percy for App sessions.